### DISCUSSION TITLE
Bayesian Neural Networks

### DISCUSSION THREAD

**Discussion by CometManAtGitHub** [MIC-DKFZ/nnUNet#660] [Ideas]
Hi all,

are there plans or offorts to expand nnUNet with regard to Bayesian Neural Networks?

Here is an example:
https://paperswithcode.com/paper/hands-on-bayesian-neural-networks-a-tutorial
https://arxiv.org/abs/2007.06823v1

This is interesting in regards to Active-Learning.

Thanks

**Comment by FabianIsensee** [comment]
Hi, this is certainly interesting but not of high priority at the moment
